sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -49,7 +49,7 @@ struct code_location {
   }
 #else
   static constexpr code_location
-  current(const char *fileName = __builtin_FILE(),
+  current(const char *fileName = nullptr,
```

Comment:
I would suggest adding TODO here that mentions what this workaround is aimed at.

Reply:
Added a `FIXME`